### DISCUSSION TITLE
Any plans in the road map to bring profiling support to on-premise applications and services?

### DISCUSSION THREAD

**Discussion by twbaltimore** [microsoft/ApplicationInsights-Profiler-AspNetCore#137] [Q&A]
We are in the process of assessing Application Insights as an APM for our applications/services.  One key missing component that would be nice to have for us is profiling, as we do not host within Azure.  Is there any plans in the profiling roadmap for Application Insights where this will be offered for applications and services not hosted in Azure?

**Comment by xiaomi7732** [comment]
@twbaltimore What is your target framework? There's no plan for .NET framework application while this profiler already supports .NET Core / .NET 5 application on premise, although an application insights resource to host the profiler file is still required.
